tests/test_generate_success.py

```diff
@@ -0,0 +1,58 @@
+# Global imports
+"""Import of pytest module for testing"""
+import pytest
+
+"""Import generate config functions test perfect scenario from that function"""
+from gatorgrade.generate.generate import generate_config
+
+
+@pytest.fixture()
+def setup(tmp_path):
+    """Setup directory for tests"""
+    # Given this file structure
+    src_dir = tmp_path / "src"
+    src_dir.mkdir()
+    python_file = src_dir / "test.py"
+    python_file.write_text("import sys")
+    writing_dir = tmp_path / "writing"
+    writing_dir.mkdir()
+    reflection = writing_dir / "reflection.md"
+    reflection.write_text("# Reflection")
+    yield tmp_path
+
+
+def test_generate_config_creates_gatorgrade_yml(setup):
+    """Test to see that gatorgrade_yml file exist in file structure"""
+    # When generate_config is called
+    generate_config(["src"], setup)
+    # Then gatorgrade.yml is created
+    gatorgrade_yml = setup / "gatorgrade.yml"
+    assert gatorgrade_yml.is_file()
+
+
+def test_generate_config_creates_gatorgrade_yml_with_dir_in_user_input(setup):
```

Comment:
Do you think having a shorter function name would improve readability? If not, this looks good!